recipes/libcurl/all/conanfile.py

```diff
@@ -379,7 +385,10 @@ def _configure_autotools(self):
         return self._autotools, self._configure_autotools_vars()
 
     def _configure_cmake(self):
-        cmake = CMake(self)
```

Comment:
Can you cache the cmake object to avoid reconfiguration in `package()` after `build()`?

Reply:
sure